actors/builtin/miner/policy.go

```diff
@@ -52,8 +56,9 @@ func loadPartitionsSectorsMax(partitionSectorCount uint64) uint64 {
 // The maximum number of new sectors that may be staged by a miner during a single proving period.
 const NewSectorsPerPeriodMax = 128 << 10
```

Comment:
**Deletion request:** Parameter is never used and should likely be removed.

Reply:
@anorth or @Stebalien should confirm there is no intention to use a limit like this before deletion